vizro-core/src/vizro/models/_action/_action.py

```diff
@@ -35,6 +36,16 @@ class Action(VizroBaseModel):
         regex="^[a-zA-Z0-9_]+[.][a-zA-Z_]+$",
     )
 
+    @validator("function")
+    def check_export_imports(cls, function):
+        if (
+            function._CapturedCallable__function.__name__ == "export_data"
+            and function._arguments.get("file_format") == "xlsx"
+        ):
+            if importlib.util.find_spec("openpyxl") is None and importlib.util.find_spec("xlsxwriter") is None:
+                raise ModuleNotFoundError("You must install either openpyxl or xlsxwriter to export to xlsx format.")
```

Comment:
```suggestion
                raise ModuleNotFoundError("Either openpyxl or xlsxwriter must be installed to export to xlsx format.")
```

Reply:
Hm, I think active is perferrable over passive no?

Reply:
I prefer the passive, because you must seems quite harsh - but i guess it is personal preference :D